src/submission/models.py

```diff
@@ -1459,6 +1459,21 @@ def full_name(self):
             full_name = "%s %s" % (full_name, self.name_suffix)
         return full_name
 
+    @property
+    def dc_name_string(self):
+        name_string = ""
+        if self.last_name:
+            name_string += "{}{}, ".format(
```

Comment:
The comma here should be added when there is a first name only, no?

We also need to consider the case for `is_corporate` (just call `self.corporate_name`)